### ISSUE TITLE
Stage weapons should fill in extra slots when using Silent Old Sanctuary

### ISSUE THREAD

**Issue by wraith309** [Danmer/vs-build-planner#42]
resulting in cases like this:
![image](https://github.com/Danmer/vs-build-planner/assets/38050849/d4fe99af-ffb9-4144-93c6-9e1b8d8cb121)

![image](https://github.com/Danmer/vs-build-planner/assets/38050849/93a2b047-586c-4597-a4a3-c8c61d142ab7)

**Comment by wraith309** [comment]
actually, upon further investigation, this isn't an issue with stage items, but one with combining union weapons with any other weapon. after all you can just grab the extra weapon _after_ the slot frees up.

![image](https://github.com/Danmer/vs-build-planner/assets/38050849/ea2ed148-96e7-4fd8-b744-f6c96f68bf0d)
![image](https://github.com/Danmer/vs-build-planner/assets/38050849/68d908d5-d446-46d2-8d8d-34e9dda40fbb)
![image](https://github.com/Danmer/vs-build-planner/assets/38050849/1c3e0480-e2c1-4408-813a-9bd0fa261fe2)
![image](https://github.com/Danmer/vs-build-planner/assets/38050849/8e47a728-8d93-4ced-8da0-9b6e02b8b79d)

basically every union looks like this.

**Comment by Danmer** [comment]
Fixing Silent Old Sanctuary is like:

![bugs](https://github.com/Danmer/vs-build-planner/assets/390169/4172dd23-d05f-4f01-bca3-c9e7627caeb5)
